### DISCUSSION TITLE
Avoid shipping OSL-3.0 code

### DISCUSSION THREAD

**Discussion by stefan6419846** [pradyunsg/furo#707] [Ideas] [answered]
At the moment, *furo* ships with an OSL-3.0 module (https://github.com/pradyunsg/furo/blob/main/src/furo/_demo_module.py) in its wheel files and distributions. Nevertheless, it only seems to be required to build the docs.

I stumbled upon this when reviewing the file license headers, as there is no corresponding hint in the actual license file. Some people (like the Free Software Foundation) advise to not use it and declare it incompatible with the GPL: https://www.gnu.org/licenses/license-list.html.en#OSL

For this reason, I would like to see the corresponding file to be omitted from distribution or re-licensed if possible to avoid unexpected side effects.

**Comment by pradyunsg** [comment, marked as answer]
079d829f
